packages/visx-demo/happo/gallery.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { tiles as examples } from '../src/components/Gallery';
+import AxisTile from '../src/components/Gallery/AxisTile';
+import XYChartTile from '../src/components/Gallery/XYChartTile';
+
+type HappoSnapshot = {
+  component: string;
+  variants: {
+    [key: string]: (
+      renderInDom: (component: React.ReactElement) => void,
+    ) => React.ReactNode | Promise<unknown>;
+  };
+};
+
+const MAX_TIMEOUT_MS = 400;
```

Comment:
Why 400? Is there a variable somewhere else that this could import and be based on?

Reply:
ideally this is in sync with `.happo.js`, will update to import from there.

Reply:
couldn't import `asyncTimeout` directly from `.happo.js` because `node`'s `child_process` gets pulled into the webpack config step which throws. put it in a separate file.